### ISSUE TITLE
Icons are funky

### ISSUE THREAD

**Issue by SolDev69** [PrismLauncher/PrismLauncher#90]
### Operating System

Winblows 11

### Version of PolyMC

7f951a05

### Version of Qt

Qt 6.4.0

### Description of bug

![image](https://user-images.githubusercontent.com/40839581/196546128-c0b8af68-af4b-4c6b-97c1-cda45b52a286.png)
Check boxes in the "edit instance"/ console window are too wide.

### Steps to reproduce

Seems to affect all instances.

### Suspected cause

Some sort of gui change made after PolyMC 1.4.2. Never got changed before the fork.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by TayouVR** [comment]
in my 1.4.2 instance it happens as well: 
![image](https://user-images.githubusercontent.com/31988415/196549344-91af422b-6ba1-4be6-818e-9788a249a231.png)

**Comment by Scrumplex** [comment]
Still happening?

**Comment by TayouVR** [comment]
yep, this seems to be related to the windowsvista theme, if i switch to fluent or the built in dark theme (fusion) it doesn't happen. So maybe we can close it as it might a qt bug... 🤔 
Though we can load a stylesheet in code on top of the system theme to correct this. But that is kinda jank...

I have multiple themes, that display it correctly and multiple that don't; if we want to investigate the qss rules and fix it ourselves...
